site/src/comparison.rs

```diff
@@ -423,6 +439,102 @@ impl ComparisonSummary {
     }
 }
 
+/// Writes a Markdown table containing summary of relevant results.
+pub fn write_summary_table(
+    primary: &ComparisonSummary,
+    secondary: &ComparisonSummary,
+    result: &mut String,
+) {
+    use std::fmt::Write;
+
+    fn render_stat<F: FnOnce() -> Option<f64>>(count: usize, calculate: F) -> String {
+        let value = if count > 0 { calculate() } else { None };
+        value
+            .map(|value| format!("{value:.1}%"))
+            .unwrap_or_else(|| "N/A".to_string())
+    }
+
+    writeln!(
+        result,
+        r#"| | Regressions 😿 <br> (primary) | Regressions 😿 <br> (secondary) | Improvements 🎉 <br> (primary) | Improvements 🎉 <br> (secondary) | All relevant primary changes |
```

Comment:
Instead of "All relevant primary changes" which renders very wide and makes the table as a whole very wide, perhaps we can do "All 😿🎉 \<br\> (primary)". That should make the table significantly less wide.